src/components/App.tsx

```diff
@@ -242,6 +253,9 @@ export class App extends React.Component<AppProps, AppState> {
       if (project.getFile("README.md")) {
         openFiles([["README.md"]]);
       }
+      if (this.state.quickStart) {
+        openFiles([["assembly/main.ts", "src/main.js"]]);
```

Comment:
let's store in fiddle which files need to be open, especially given `fiddle` is available here

Reply:
Adding a todo to address in a diff pull request. I think that this is a good enough first stab.

Reply:
Let's try to put things into https://github.com/nearprotocol/NEARStudio/issues and move into appropriate milestones.